app/src/main/java/com/example/music_app/ui/screens/playlists/PlaylistsField.kt

```diff
@@ -34,30 +33,37 @@ fun PlaylistsField(
             .clip(RoundedCornerShape(AppTheme.dimens.spacing08))
             .background(MaterialTheme.colorScheme.primaryContainer)
     ) {
-        if (isLoading) {
-            CircularProgressIndicator(
-                color = Color.Blue,
-                modifier = Modifier
-                    .size(AppTheme.dimens.spacing80)
-            )
```

Comment:
I think previous solution was fine and could remain here for initial loading, bottom item solution is more suitable only for pagination loading indication. Could you try to have both indications for different occasions ?